src/Castle.Windsor/MicroKernel/Registration/Component.cs

```diff
@@ -34,7 +34,7 @@ public static ComponentRegistration For(Type serviceType)
 		{
 			if (serviceType == null)
 			{
-				throw new ArgumentNullException("serviceType",
+				throw new ArgumentNullException(nameof(serviceType),
```

Comment:
Could this be:

`"The serviceType argument was null. Check that the assembly "
				                                + "is referenced and the type available to your application.");					                                + "is referenced and the type available to your application.");`